src/common/transformer/test/test.server.js

```diff
@@ -11,8 +11,9 @@ try {
     Builder = require('marko/compiler/Builder');
 } catch (e) {
     // v4 paths
-    CompileContext = require('marko/dist/compiler/CompileContext');
-    Builder = require('marko/dist/compiler/Builder');
+    const target = require('marko/env').isDebug ? 'src' : 'dist';
+    CompileContext = require(`marko/${target}/compiler/CompileContext`);
+    Builder = require(`marko/${target}/compiler/Builder`);
```

Comment:
Are we testing that this compiles properly in v3?

Reply:
@mlrawlings ☝️ ?

Reply:
I am assuming it does compile properly, given that it passes the v3 and v4 integration tests.